src/genericTensorOps.hpp

```diff
@@ -539,6 +563,28 @@ void add( DST_MATRIX && LVARRAY_RESTRICT_REF dstMatrix,
   }
 }
 
+/**
+ * @brief Subtract @p value to @p dstVector.
+ * @tparam M The length of @p dstVector.
+ * @tparam DST_VECTOR The type of @p dstVector.
+ * @param dstVector The destination vector, of length M.
+ * @param value The value to subtract.
+ * @details Performs the operation @code dstVector[ i ] -= value @endcode
+ */
+template< std::ptrdiff_t M, typename DST_VECTOR >
+LVARRAY_HOST_DEVICE CONSTEXPR_WITHOUT_BOUNDS_CHECK inline
+void subtractScalar( DST_VECTOR && LVARRAY_RESTRICT_REF dstVector,
```

Comment:
I'm guessing you think it's worth having both `addScalar` and `subtractScalar`. Is there a reason not to have `subtractScalar` call `addScalar< M >( dstVector, -value )`?

Reply:
No, it's just me being on autopilot :see_no_evil: